conf/app.routes

```diff
@@ -3,3 +3,7 @@
 GET        /language/:lang          controllers.LanguageController.switchToLanguage(lang: String)
 GET        /sign-out                controllers.SignOutController.signOut
 GET        /hello-world             controllers.HelloWorldController.helloWorld
+
+# Confirm Email
+GET         /confirm-email          controllers.ConfirmEmailController.show
+GET         /confirm-email          controllers.ConfirmEmailController.updateEmail
```

Comment:
Is this supposed to be a POST? Or is this being handled in another task?

Reply:
As we are retrieving the email address from session, this doesn't need to be a post. Confirmed this with Gary on Friday.